app/views/list_responses/feedbacks.html.slim

```diff
@@ -0,0 +1,66 @@
+= render 'reminder_mailings/section_navigation'
+
+h1 Eingereichte Feedbacks
+
+nav.navbar.section-navigation.hidden-print
+  ul.list-inline
+    li= button_link 'Filter aufheben', url_for(q: { s: ['updated_at asc'] }), dimension: :sm
```

Comment:
AFAIK, they only requested to have lists of the unreviewed feedbacks, unreviewed trial feedbacks and unpaid hours.